runtime/common/src/lib.rs

```diff
@@ -276,9 +276,17 @@ parameter_types! {
 // pallet liquidity mining
 parameter_types! {
 	pub const LMPalletId: PalletId = PalletId(*b"LiqMinId");
-	pub const MinPlannedYieldingPeriods: BlockNumber = 100;
-	pub const MinTotalFarmRewards: Balance = NATIVE_EXISTENTIAL_DEPOSIT * 1_000;
-	pub const NftClass: primitives::CollectionId = 1;
+	pub const LiquidityMiningNftCollectionId: primitives::CollectionId = 1;
+}
+
+// warehouse pallet liquidity mining
+parameter_types! {
+	pub const WarehouseLMPalletId: PalletId = PalletId(*b"WhouseLm");
+	pub const MinDeposit: Balance = 1;//TODO: tmp value
+	pub const MaxEntriesPerDeposit: u8 = 5; //NOTE: Rebenchmark when this change //TODO: tmp value
+	pub const MaxYieldFarmsPerGlobalFarm: u8 = 5; //TODO: tmp value
+	pub const MinPlannedYieldingPeriods: BlockNumber = 100;  //TODO: tmp value
+	pub const MinTotalFarmRewards: Balance = NATIVE_EXISTENTIAL_DEPOSIT * 1_000; //TODO: tmp value
```

Comment:
TODO

Reply:
This is for @jak-pan. I don't know what values we want